Create Local Legend(s) / 地方伝説 / 地方传奇 / 地方傳奇 and Add new keywords for Version 5.8 sequel genshin-langdata

yasuking0304 opened the pull request:
1. Create a tag for the Local Legend(s) / 地方伝説 / 地方传奇 / 地方傳奇
2. Since we have advanced the story of the 2nd half of Ver 5.8, we will add keywords. This time, I am creating it from Genshin Impact's item. movie, youtube. bilibili.

Closes #434
Closes #436

Tags
| key | English | Japanese | Simplified | Traditional | Title(en) | Title(ja) | Title(zh-cn) | Title(zh-tw) |
|---|---|---|---|---|---|---|---|---|
| enemy-legend | Local Legend | 地方伝説 | 地方传奇 | 地方傳奇 | Chinese & Japanese translations for Local Legends in Genshin Impact | 原神の地方伝説の英語・中国語表記一覧 | 原神中地方传奇的英语和日语翻译 | 原神中地方傳奇的英語和日語翻譯

Living
| English | Japanese | Simplified | Traditional | Note(en) | Note(ja) | Note(zh-cn) | memo|
|---|---|---|---|---|---|---|---|
| Ancient Glazeback Turtle | 琉彩ガメ・古種 | 琉背古彩龟 | 琉背古彩龜

Ememies
| English | Japanese | Simplified | Traditional | Note(en) | Note(ja) | Note(zh-cn) | memo|
|---|---|---|---|---|---|---|---|
| local legend(入力済み) | 地方伝説 | - | - | Legendary enemies scattered throughout Teyvat | テイワット内に点在する伝説の敵 |  | remove tags:[fontainue] and add tags:[enemy-legend]
| Vivianne of the Lake(入力済み) | 湖辺のヴィヴィアン | - | - | | | | add tags:[ enemy-legend]
| Ninianne of the Lake(入力済み) | 湖辺のニニアン | - | - | | | | add tags:[ enemy-legend]
| Ocean Circuit Judge(入力済み) | オーシャン巡回法官 | - | - | | | | add tags:[ enemy-legend]
| Iron Viscount(入力済み) | 鉄皮の子爵 | - | - | | | | add tags:[ enemy-legend]
| Angelica, Fairy Knight Twin(入力済み) | 双生妖精の騎士・アンジェリカ | - | - | | | | add tags:[enemy-legend]
| Medoro, Fairy Knight Twin(入力済み) | 双生妖精の騎士・ミドロ | - | - | | | | add tags:[enemy-legend]
| Cortana, Sword of the Gorge(入力済み) | 深谷の剣・コルタナ | - | - | | | | add tags:[enemy-legend]
| Murgleis, Sword of the Gorge(入力済み) | 深谷の剣・モーグレイ | - | - | | | | add tags:[enemy-legend]
| Fading Veteran(入力済み) | 「消えゆく老兵」 | - | - | | | | add tags:[enemy-legend]
| Dobharcu, Lord of the Hidden(入力済み) | 隠者の王・ドアーチュ | - | - | | | | add tags:[enemy-legend]
| Polychrome Tri-Stars |「 異色三連星」 | 「异色三连星」 | 「異色三連星」| | ナタの地方伝説の3人のファデュイ | | add tags:[enemy, fatui, natlan, enemy-legend]
| Cocijo | 「コシーホ」 | 「科西霍」 | 「科西霍」| | ナタの地方伝説の1体で、琉彩ガメの変異体| | add tags:[natlan,  living, enemy, enemy-legend]
| Sappho Amidst the Waves | 「波のジャプー」 | 「海浪中的莎孚」 | 「海浪中的莎芙」| | ナタの地方伝説の1人の人物 | | add tags:[character-sub, natlan, enemy-legend]
| Balachko | 「バラチコ」 | 「巴拉奇科」 | 「巴拉奇科」| | ナタの地方伝説の1人のファデュイ | | add tags:[character-sub, fatui, natlan, enemy-legend]
| Cihuacoat l of Chimeric Bone | 「神骨の蛇姫」 | 「嵌合神骨的蛇姬」 | 「嵌合神骨的蛇姬」| | ナタの地方伝説の1人の人物 | | add tags:[character-sub, natlan, enemy-legend]
| He Never Dies | 「アンデッド」 | 「不死」 | 「不死」| | ナタの地方伝説の1体で、カピバラの霊体 | | add tags:[natlan, enemy, enemy-legend]
| Chimalli's Shade | 「シェマリの陰」 | 「切玛利的遮荫」 |  「切瑪利的遮蔭」| | ナタの地方伝説の1体で、大霊の化身の変異体 | | add tags:[natlan, enemy, enemy-legend]
| Atlatl's Blessing | 「アトラトの祝福」 | 「阿特拉托的佑助」| 「阿特拉托的佑助」| | ナタの地方伝説の1体で、大霊の化身の変異体 | | add tags:[natlan, enemy, enemy-legend]
| Rilai | リライ | 莉莱 | 莉萊 | | ナタの地方伝説1人の人物 | | add tags:[character-sub, natlan, enemy-legend]
| "Spirit of the Fallen Dawnstar" | 「墜落せし晨星の霊」 | 「陨落的晨星之灵」 | 「隕落的晨星之靈」| | ナタの地方伝説の1体で、大霊の化身の変異体 | | add tags:[natlan, enemy, enemy-legend]
| "Bronzelock" | 「銅の錠」| 「铜锁」 | 「銅鎖」 | | ナタの地方伝説の1体で、遺跡ドレイク・陸行の変異体 | | add tags:[natlan, enemy, enemy-legend]
| "Potapo's Solidarity" | 「ポタポの応援」 | 「波塔波的声援」 | 「波塔波的聲援」| | ナタの地方伝説の1体で、大霊の化身の変異体 | | add tags:[natlan, enemy-legend]
| Ironbeard | 鉄髭 | 铁须 | 鐵鬚 | | ナタの地方伝説の1人の人物 | | add tags:[character-sub, natlan, enemy-legend]
| "Tupu's Lushness" | 「トゥプの繁栄」| 「图普的繁茂」 | 「圖普的繁茂」| | ナタの地方伝説の1体で、大霊の化身の変異体 | | add tags:[natlan, enemy, enemy-legend]
| "Reji" | 「レジ」 | 「雷吉」 | 「雷吉」| | ナタの地方伝説の1人の人物 | | add tags:[character-sub, natlan, enemy, enemy-legend]
| "Infinitesimal" | 「微末」　| 「微末」 | 「微末」 | | ナタの地方伝説の1体で、溶岩の像の変異体 | | add tags:[natlan, enemy, enemy-legend]
| The Peak | ピーク | 尖峰 | 尖峰 | | ナタの地方伝説の1体で、ライノ竜の変異体 | | add tags:[natlan, living, enemy, enemy-legend]
| "Battlegoat" | 「戦羊」 | 「战羊」 | 「戰羊」 | | ナタの地方伝説の1体で、ショコアトルヤギの変異体 | | add tags:[natlan, living, enemy, enemy-legend]
| "Iconclaw" | 「鉄爪」|「铁爪」 | 「鐵爪」 | | ナタの地方伝説の1体で、浮燃ペンギンの変異体 | | add tags:[natlan, living, enemy, enemy-legend]
| Capybara Great Majesty!(入力済み) | カピバラ キング! | - | - | | | | add tags:[enemy-legend]

Events
| English | Japanese | Simplified | Traditional | Note(en) | Note(ja) | Note(zh-cn) | memo|
|---|---|---|---|---|---|---|---|
| Mementos of Teyvat: Prelude of the Frozen Veil | テイワットの思い出・銀世界へのプレリュード | 提瓦特忆画・雪幕前事 | 提瓦特憶畫・雪幕前事 | Limited Time Event in v5.8 | v5.8 期間限定イベント | | add tags:[events]
| Tracing Vanishing Trails | 密かな追跡 | 谛踪匿行 | 諦蹤匿行 |Limited Time Event in v5.8 | v5.8 期間限定イベント | | add tags:[liyue, events]

World Quest
| English | Japanese | Simplified | Traditional | Note(en) | Note(ja) | Note(zh-cn) | memo|
|---|---|---|---|---|---|---|---|
| Convert Investigation, Dirty Money Misdeeds  | ハラハラ潜入捜査 | 隠行谛勘, 秽金之罪 | 隠行諦勘, 穢金之罪 | | | | add tags:[event]


Character (If there is no additional note, tags: [character-sub])
| English | Japanese | Simplified | Traditional | Note(en)  | Note(ja) | Note(zh-cn)  | memo |
|---|---|---|---|---|---|---|---|
| Edler | エドラー | 埃德勒 | 埃德勒 |  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[fontaine, liyue]
| Corder | コーダー | 科德 | 科德 |  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[fontaine, liyue] 
| Doddamani | ドッダマーニ | 多达马尼 | 多達馬尼|  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[sumeru, fontaine, liyue]
| Scamface | 邪知深き狂者 | 诈狂 | 詐狂  |  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[how-to-call]
| Diderot | ディーサー | 笛特 | 笛特 |  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[fontaine]
| Rovere Pellicia | ロベレ・ペリシア | 罗韦雷・佩里恰 | 羅韋雷・佩里恰 |  | v5.8 期間限定イベント「密かな追跡」に登場する人物 | | add tags:[fontaine]

Organization
| English | Japanese | Simplified | Traditional | Note(en)  | Note(ja) | Note(zh-cn)  | memo |
|---|---|---|---|---|---|---|---|
| Dark Financier | ブラックマネー・ロード | 黑钱领主 | 黑錢領主 |  | v5.8 期間限定イベント「密かな追跡」に登場する組織 | | add tags:[event, organization]

Items
| English | Japanese | Simplified | Traditional | Note(en)  | Note(ja) | Note(zh-cn)  | memo |
|---|---|---|---|---|---|---|---|
| Forensic Imaging Device | 総合映像記録器 | 综像勘录仪 | 綜像勘録儀 || | | add tags:[event]
| Invitation to an Auction | 競売の招待状 | 拍卖会的邀请函 | 拍賣會的邀請函 |  | | | add tags:[event]

xicri commented on dataset/dictionary/enemies.ts:
```diff
@@ -2434,22 +2443,13 @@ export default [
     zhTW: "湖畔的維維安涅",
     pronunciationJa: "こへんのヴィヴィアン",
-    tags: [ "fontaine", "enemy" ],
+    tags: [ "fontaine", "enemy", "enemy-legend" ],
```
Looks like there are Local Legends only with `enemy-legend` tag and the ones with both `enemy` and `enemy-legend` tags.
I think we should make them consistent.

Which do you prefer, adding `enemy` tag or not?
I considered to replace `enemy` tags with `enemy-legend` tags from the Local Legends because the Bosses does not have `enemy` tag.
However, I don't have any strong opinion for this and adding both `enemy` and `enemy-legend` is also fine for me.

yasuking0304 commented:
Add 'enemy' tag to Natlan legend entries
1. Updated tags for several Natlan local legend entries to include the 'enemy' tag for consistency.
2. Also fixed Japanese spacing in 'Polychrome Tri-Stars' entry.

xicri reviewed (approved):
Thanks, looks good!
